compose/material3/material3/api/material3.klib.api

```diff
@@ -11,14 +11,6 @@ open annotation class androidx.compose.material3/ExperimentalMaterial3Api : kotl
     constructor <init>() // androidx.compose.material3/ExperimentalMaterial3Api.<init>|<init>(){}[0]
 }
 
-open annotation class androidx.compose.material3/ExperimentalMaterial3ComponentOverrideApi : kotlin/Annotation { // androidx.compose.material3/ExperimentalMaterial3ComponentOverrideApi|null[0]
```

Comment:
Why there i adding this annotation in a cherry-pick commit, and removing from this commit?

Was there an amend in the cherry-pick commit?

Reply:
Please make this PR to jb-main if [this is true](https://github.com/JetBrains/compose-multiplatform-core/pull/2295#discussion_r2260663677)